ocdaction/ocdaction/templates/challenge/challenge_list.html

```diff
@@ -51,28 +45,27 @@ <h3 class="text-align-center">In progress</h3>
                 <div class="col-lg-12">
                     <div class="ibox float-e-margins">
                         <div class="ibox-title">
-                            <h5 class="h3 text-align-center">My Challenges</h5>
+                            <h3 class="text-align-center">My Challenges</h3>
                             <div class="ibox-tools">
-                                <a class="close-link">
-                                    <i class="fa fa-times"></i>
+                                <a class="button button--secondary" href="{% url 'challenge-add' %}">
+                                    <span class="glyphicon glyphicon-plus-sign margin-right-5"></span>Add
                                 </a>
                             </div>
                         </div>
                         <div class="ibox-content">
                             <table class="table table-striped">
                                 <thead>
                                 <tr>
-                                    <th>Level</th>
-                                    <th>Challenge</th>
-                                    <th></th>
+                                    <th class="ibox-content bold">Level</th>
+                                    <th class="ibox-content bold">Challenge</th>
                                 </tr>
                                 </thead>
                                 <tbody>
                                     {% for challenge in challenges_not_in_progress %}
                                         <tr>
-                                            <td class="text-align-left">{% get_anxiety_level challenge %}</td>
-                                            <td class="text-align-left"><a href="{% url 'challenge' challenge.id %}">{{ challenge.challenge_name }}</a></td>
-                                            <td><a class="" href="">Do challenge</a></td>
+                                            <td class="text-align-left ibox-content">{% get_anxiety_level challenge %}</td>
+                                            <td class="text-align-left ibox-content"><a href="{% url 'challenge' challenge.id %}">{{ challenge.challenge_name | title }}</a></td>
+                                            <td class="ibox-content text-align-right"><a class="" href="{% url 'challenge-score-form-new' challenge.id %}"><span class="glyphicon glyphicon-play-circle margin-right-5"></span>Do the challenge</a></td>
                                         </tr>
                                     {% endfor %}
                                 </tbody>
```

Comment:
I have added a comment to the previous commit to highlight the code that was removed but which is required. For the challenges not in progress it should check whether another challenge is in progress.  If there is another challenge in progress then the user cannot start another challenge and the do the challenge button is disabled for all the other challenges.
See PR #112